src/metadata.ts

```diff
@@ -228,7 +228,29 @@ export const applyMetadata = async (): Promise<void> => {
   logger.info('Applying metadata...');
 
   logger.debug('Reloading metadata...');
-  await reloadMetadata();
+  const { is_consistent, inconsistent_objects } = await reloadMetadata();
+
+  // * Manage metadata inconsistencies
+  if (is_consistent === false) {
+    // * Drop inconsistencies related to the change from `authenticators` to `security_keys`
+    // * Will drop if and only if the inconsistency is related to this name change
+    if (
+      inconsistent_objects?.length === 2 &&
```

Comment:
a race condition (parallel deployments of other services, OOM Kill, etc.) may happen between lines 231 and 234, so in my opinion this fix doesn't help in all cases

Reply:
scratch that